docs/src/main/sphinx/release/release-355.md

```diff
@@ -0,0 +1,33 @@
+# Release 355 (XX Apr 2021)
+
+## General
+
+* Add support for configuring multiple password authentication plugins. ({issue}`7151`)
+* Do not allow inserting data into a table that has a row filter. ({issue}`7346`)
+* Rename `optimizer.plan-with-table-node-partitioning` to `optimizer.use-table-scan-node-partitioning`. ({issue}`7257`)
+* Planning with table scan node partitioning is now triggered automatically
+  when ratio between table buckets and possible table scan tasks exceeds or is equal to
+  `optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio`. This helps to improve
+  query parallelism when table bucket count is small compared to number of nodes. ({issue}`7257`)
+* Include information about {doc}`/admin/spill` in {doc}`/sql/explain-analyze`. ({issue}`7427`)
+* Fix inconsistent behavior for {func}`to_unixtime` with values of type `timestamp(p)`. ({issue}`7450`)
+* Change return type of {func}`from_unixtime` and {func}`from_unixtime_nanos` to `timestamp(p) with time zone`. ({issue}`7460`)
+* Report tables referenced in materialized views to even listener. ({issue}`7483`)
+* Report columns which are created/inserted/updated to event listener. ({issue}`7425`)
```

Comment:
"created, inserted or updated"

also, would be good to mention where this information can be seen (`QueryCompletedEvent > ioMetadata > output > columns`)